### DISCUSSION TITLE
#不   登   校

### DISCUSSION THREAD

**Discussion by canifish** [share121/inter-knot#1658] [其他]
![IMG_20240812_140007](https://github.com/user-attachments/assets/8dd6aa9b-bf4f-47b4-8eed-2c32be5ad834)
![Screenshot_2024-07-29-16-05-48-507_com schunshang bij cbl](https://github.com/user-attachments/assets/f00f6f28-b5da-4a5b-be59-6b00bc73d0bc)
苦亚西苦亚西苦亚西，看完京吹三深感愤怒无奈，决定不登校以缓解激动的情绪

**Comment by canifish** [comment]
其实就是我买的日版t恤到了😃😃😃

**Comment by BaCktOZwero** [comment]
我买到盗版了😃😃😃
不过质量还不错（）

**Comment by JIUYINGSHI** [comment]
话说这个t恤如果不是漫展我还真不太敢穿出去

**Reply by canifish** [reply]
上学穿（doge）

**Comment by zhangyj0523** [comment]
好巧，买了同款
